lisa/microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -1057,6 +992,10 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self.repo_root = tool_path / "cloud-hypervisor"
         self.cmd_path = self.repo_root / "scripts" / "dev_cli.sh"
 
+        # Pass through MIGRATABLE_VERSION from pipeline environment if set
+        if "MIGRATABLE_VERSION" in os.environ:
+            self.env_vars["MIGRATABLE_VERSION"] = os.environ["MIGRATABLE_VERSION"]
+
```

Comment:
What's the usage of storing `MIGRATABLE_VERSION:-not_set`?

Reply:
This logs the Cloud Hypervisor version being tested at the start of each test run. The :-not_set bash parameter expansion means:

If MIGRATABLE_VERSION is set → print its value
If MIGRATABLE_VERSION is not set → print "not_set"